gluten-core/src/main/scala/org/apache/spark/util/memory/TaskMemoryResources.scala

```diff
@@ -65,7 +65,8 @@ object TaskMemoryResources extends Logging {
           // in case of crashing in task completion listener, errors may be swallowed
           new TaskFailureListener {
             override def onTaskFailure(context: TaskContext, error: Throwable): Unit = {
-              logError(s"Task ${context.taskAttemptId()} failed by error: ", error)
+              // logError(s"Task ${context.taskAttemptId()} failed by error: ", error)
+              // The general duty of printing error message should not reside in memory module
```

Comment:
Any reason removing this log?

The log was added because if the process is crashed (e.g. sigabrt) in TaskCompletionListener, and there was an exception caused the task to be completed, then the exception will not be logged.

You could refer to Spark's code:
https://github.com/apache/spark/blob/6cd9d88e2379085595e87a58de0a0c3409cfeb88/core/src/main/scala/org/apache/spark/TaskContext.scala#L159-L182

Reply:
after offline discussion, we agree to enable the log and fix this issue in the future